Writing-state-migrations.md

```diff
@@ -51,7 +51,7 @@ If your PR changes the properties of an exploration or state (or other structure
      - core/templates/services/state-top-answers-stats.service.spec.ts
```

Comment:
Should the list of files be removed?

Reply:
No, These files may need change according to the requirements. I think this list should be there.

Reply:
I do not remember you changing those files, also I think a list can be easily outdated, I would rather remove it and explain what needs to be done.

Reply:
I have removed the list of files and added the description.